src/compo/modis_aod2ioda.py

```diff
@@ -149,10 +178,10 @@ def _read(self):
 def main():
 
     # get command line arguments
-    # Usage: python blah.py -i /path/to/obs/2021060801.nc /path/to/obs/2021060802.nc ... -p <Terra or Aqua> -t Analysis_time /path/to/obs/2021060823.nc
-    # -o /path/to/ioda/20210608.nc
-    # where the input obs could be for any desired interval to concatenated together. Analysis time is generally the midpoint of
-    # analysis window.
+    # Usage: python blah.py -i /path/to/obs/2021060801.nc /path/to/obs/2021060802.nc ... -p <Terra or Aqua>
+    # -o /path/to/ioda/20210608.nc --date_range YYYYMMDDHH YYYYMMDDHH
```

Comment:
Will there ever be a need to convert this at smaller intervals if MODIS data is every 5 minutes?